setup.py

```diff
@@ -1,42 +1,41 @@
-# Copyright (c) 2020 Aiven, Helsinki, Finland. https://aiven.io/
-
-from importlib.machinery import SourceFileLoader
-from setuptools import find_packages, setup
-
-import sys
-
-
-def get_version():
-    return SourceFileLoader("version", "aiven_db_migrate/migrate/version.py").load_module().__version__
-
+from distutils.core import setup
 
 setup(
-    author="Aiven",
-    author_email="support@aiven.io",
-    entry_points={
-        "console_scripts": [
-            "pg_migrate = aiven_db_migrate.migrate.pgmigrate:main",
-        ],
-    },
+    name='aiven_db_migrate',
+    packages=['aiven_db_migrate'],
+    version='1.0.0',
+    license='apache-2.0',
```

Comment:
You can drop this and use the classifier instead, as it's a well-known license.
https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#license
`License :: OSI Approved :: Apache Software License`